pkg/addon/quota.go

```diff
@@ -310,79 +310,79 @@ var quotaConfig string = `
   }
 },
 {
-  "name": "1",
-  "rate-limiting": {
-	  "unit": "minute",
-	  "requests_per_unit": 694,
-	  "alert_limits": []
-  },
-  "resources": {
-	  "backend_listener": {
-		  "replicas": 1,
-		  "resources": {
-			  "requests": {
-				  "cpu": 0.1,
-				  "memory": "450Mi"
-			  },
-			  "limits": {
-				  "cpu": 0.11,
-				  "memory": "500Mi"
-			  }
-		  }
-	  },
-	  "backend_worker": {
-		  "replicas": 1,
-		  "resources": {
-			  "requests": {
-				  "cpu": 0.03,
-				  "memory": "55Mi"
-			  },
-			  "limits": {
-				  "cpu": 0.04,
-				  "memory": "60Mi"
-			  }
-		  }
-	  },
-	  "apicast_production": {
-		  "replicas": 1,
+	"name": "1",
+	"rate-limiting": {
+		"unit": "minute",
+		"requests_per_unit": 694,
+		"alert_limits": []
+	},
+	"resources": {
+		"backend_listener": {
+			"replicas": 2,
+			"resources": {
+				"requests": {
+					"cpu": 0.06,
+					"memory": "250Mi"
+				},
+				"limits": {
+					"cpu": 0.18,
+					"memory": "300Mi"
+				}
+			}
+		},
```

Comment:
```suggestion
		"backend_listener": {
			"replicas": 2,
			"resources": {
				"requests": {
					"cpu": 0.06,
					"memory": "450Mi"
				},
				"limits": {
					"cpu": 0.18,
					"memory": "500Mi"
				}
			}
		},
```
Memory values don't seem to match ticket / table in PR for backend listener ? :thinking:

Reply:
I had one job! Good catch, thanks Kevin, I'll fix it now :D